python/schedule.hpp

```diff
@@ -18,6 +18,7 @@ struct schedule_shim_base {
     schedule_shim_base() = default;
     schedule_shim_base(const schedule_shim_base&) = delete;
     schedule_shim_base& operator=(schedule_shim_base&) = delete;
+    virtual ~schedule_shim_base() {}
```

Comment:
I'm pretty sure this line is all we need for getting the virtual destructor right; no need to implement overloads in the derived classes.

Reply:
👍